community/cypher/cypher/src/main/scala/org/neo4j/cypher/ExtendedExecutionResult.scala

```diff
@@ -19,10 +19,50 @@
  */
 package org.neo4j.cypher
 
-import org.neo4j.graphdb.{Notification, QueryExecutionType}
+import org.neo4j.graphdb.Result.{ResultRow, ResultVisitor}
+import org.neo4j.graphdb._
 
 trait ExtendedExecutionResult extends ExecutionResult {
   def planDescriptionRequested: Boolean
   def executionType: QueryExecutionType
   def notifications: Iterable[Notification]
+
+  def accept(visitor: ResultVisitor): Unit = {
```

Comment:
I think that this should be pushed even further down. Having this implementation in `PipeExecutionResult` and `ExplainExecutionResult` instead.